.eslintrc.js

```diff
@@ -32,6 +34,7 @@ module.exports = {
     'no-mixed-spaces-and-tabs': ['error'],
     'no-prototype-builtins': ['warn'],
     'no-redeclare': ['warn'],
+    'no-sync': ['warn'],
```

Comment:
This check is deprecated. I know that it is only warn but I'm bot sure we are ready to add that rule

Reply:
We can add it in next iteration